src/NUnitFramework/framework/Attributes/TestCaseAttribute.cs

```diff
@@ -426,7 +423,7 @@ public IEnumerable<TestMethod> BuildFrom(IMethodInfo method, Test? suite)
                 if (_untilDate > DateTimeOffset.UtcNow)
                 {
                     test.RunState = RunState.Ignored;
-                    string reason = string.Format("Ignoring until {0}. {1}", _untilDate.Value.ToString("u"), IgnoreReason);
+                    string reason = $"Ignoring until {_untilDate.Value.ToString("u")}. {IgnoreReason}";
```

Comment:
I helped with a VS light bulb suggestion that should be offering to change to:
```suggestion
                    string reason = $"Ignoring until {_untilDate.Value:u}. {IgnoreReason}";
```